directory/setup.py

```diff
@@ -22,6 +23,16 @@ def get_requirements(fpath):
         return f.readlines()
 
 
+manifest_version = None
+with open(path.join(HERE, 'manifest.json'), encoding='utf-8') as f:
+    manifest = json.load(f)
+    manifest_version = manifest.get('version')
+
+if ABOUT["__version__"] != manifest_version:
+    raise Exception(
+        "Inconsistent versioning in module and manifest - aborting wheel build"
+    )
+
```

Comment:
let's fix the `license` and remove `extras_require`. There's a typo in `packages` pointing to the directory check. `datadog_agent_tk` is mentioned in `tests_require` but it doesn't exist anymore. `test_suite` can go and `package_data` points to the `directory` check

Reply:
This one is actually for the directory check. Was trying to cleanup some issues from my last PR :) Addressing other comments though, thanks for those!

Reply:
lol, didn't notice the filename, sorry for the noise!